### ISSUE TITLE
Clarify licensing of schema without visual design

### ISSUE THREAD

**Issue by opatut** [openmaptiles/openmaptiles#1318]
Hello, I'd love to get a clarification for the following parts of the license file:

> The cartography and visual design features of the map tile schema (also known as the "look and feel" of the map) are licensed under the Creative Commons Attribution 4.0 license. To view a copy of the license, visit http://creativecommons.org/licenses/by/4.0/.
>
> Products or services using maps derived from OpenMapTiles schema need to visibly credit "OpenMapTiles.org" or reference "OpenMapTiles" with a link to http://openmaptiles.org/.

I may not be a lawyer, but I've been around in the open source world for a while. It reads to me like this:

* The styles based on the OpenMapTiles schema, such as [positron-gl-style](https://github.com/openmaptiles/positron-gl-style) etc., are licensed under CC-BY. This is totally reasonable, even great, as that is a very permissive license.
* The authors consider the *file format* (i.e. schema) to also be under that license. That's why there is a difference between the map styles' license file and this repo's license, namely the "cartography and" part in the first sentence.

Now I am wondering about the latter part:

* It's not CC-BY-SA, so if I adapt the schema a little, I can rename the project, citing it as being derived from OpenMapTiles, then drop the attribution on the maps derived from it. Again, I'm not a lawyer, and that sounds rather unfair to the great amount of work that was put into this project, but possible.
* The schema is basically a file format. As far as I understand, a file format cannot reasonably be protected by a license. By patents, yes, but not by copyright. Licensing works for copyrighted material, which at least in Germany is not possible with anything you produce. There is a great analogy of a cooking recipe here: As far as I know, you cannot copyright or license a cooking recipe that you write, well, you can, but only the wording and possibly images and layout are considered under the "Urheberrecht" (German copyright law). So if I write another recipe with exactly the same instructions as yours, just using my own words, that is *not* a copyright infringement. Similarly, another tool writing a file compatible with the OMT styles (such as [flatmap](https://github.com/onthegomap/flatmap/)) would IMO not be required to license the format, *and* if I use flatmap and my own style, I wouldn't be using any material copyrighted by OMT -- at least that's how I understand it, correct me if I'm wrong.

I don't want to take away attribution where appropriate here, I'm just wondering how this license-for-the-schema is legally supposed to work and whether it's airtight ;) And in the end, I think it would be reasonable to drop the license for the OpenMapTiles schema and make it public domain without restrictions, while of course keeping the licenses on the provided default styles (Positron et al.), the reference implementation (openmaptiles-tools), and all other actual software or art/design products. That would make me, as a user of the schema, sleep better at night, knowing what I can and can't do with it. I don't see how that would weaken OMT's (or MapTiler's) position anyhow, since:

- The schema is referred to as "OpenMapTiles schema" by everybody writing software compatible with it, so nobody will forget where it comes from. Kind of like `.docx` is actually an [open standard](https://en.wikipedia.org/wiki/Office_Open_XML) and everybody still calls it a "Word Document" based on the well-known product by Microsoft.
- The default styles are so good and almost everybody will base their work on them, requiring the copyright attribution on the map they produce with it due to using the style, not the schema.

Of course I might be reading the license wrong and all of this might be futile. Maybe the license file is trying to apply CC-BY to the schema *definition files*, as in the YAML and SQL files in this repository -- similar to the wording of the recipe, but not the instructions -- and hence would not apply to a tool that writes the same data, but doesn't require YAML or SQL files to do so. But then, I'm wondering what the BSD 3-clause is there for :D 

TL;DR: I am very confused by what "the cartography [...] features" in the license file refers to and how that can be licensed as CC-BY, and ask myself whether it even should be.

Thanks.

**Comment by opatut** [comment]
I see that the README has changed to:

> All code in this repository is under the [BSD license](https://github.com/openmaptiles/openmaptiles/blob/master/LICENSE.md) and the cartography decisions encoded in the schema and SQL are licensed under [CC-BY](https://github.com/openmaptiles/openmaptiles/blob/master/LICENSE.md).

This is way clearer to me.

From my understanding, when I build an application that uses openmaptiles-tools (MIT license) and a complete custom schema definition (YAML and SQL files) with no code taken from this repository in particular (such as its schema, or quickstart.sh, or any similar auxiliary files), then only the MIT license of openmaptiles-tools applies.

Thanks for the clarification!
